pipeline/00-ingest.R

```diff
@@ -147,11 +150,11 @@ recode_column_type <- function(col, col_name, dict = col_type_dict) {
     pull(var_type)
 
   switch(col_type,
```

Comment:
```suggestion
  switch(
    col_type,
```

Reply:
Other way, it wants the file on the same line, was pinging it since it wasn't in the function I created.